src/common/DOMWorld.ts

```diff
@@ -96,15 +98,17 @@ export class DOMWorld {
     `${name}_${contextId}`;
 
   constructor(
+    client: CDPSession,
     frameManager: FrameManager,
     frame: Frame,
     timeoutSettings: TimeoutSettings
   ) {
+    this._client = client;
```

Comment:
can we/should we get the client from FrameManager? or are those different clients?

Reply:
The FrameManagers client is pointing to the main target whereas a DomWorld's target might either be the mainTarget or the target of an OOP iframe.

Reply:
is it worth leaving a comment about that?